package.json

```diff
@@ -53,10 +54,12 @@
     "form-data": "^2.3.1",
     "go-ipfs-dep": "^0.4.13",
     "hat": "0.0.3",
-    "ipfs": "~0.27.7",
+    "ipfs": "^0.28.0",
```

Comment:
Keep using the `~`.

Reply:
npm install annoyingly changes the `^` and `~` :(

Reply:
I know, you just need to check on submitting the PR. Reviewing one's PRs can go a long way.